bootstrap.sh

```diff
@@ -17,6 +18,32 @@ if [ ! -f killrvideo_bootstrapped ]; then
   fi
   echo "=> Setting up KillrVideo via DSE node at: $dse_ip"
 
+  # If a request timeout is available use that.  This is useful
+  # in cases where a longer timeout is needed for cqlsh operations
+  if [ ! -z "$KILLRVIDEO_DSE_REQUEST_TIMEOUT" ]; then
+    dse_request_timeout="--request-timeout=$KILLRVIDEO_DSE_REQUEST_TIMEOUT --connect-timeout=$KILLRVIDEO_DSE_REQUEST_TIMEOUT"
+
+    echo "=> Request timeout set at: $dse_request_timeout"
+  fi
+
+  # If SSL is enabled, then provide SSL info
+  if [ "$KILLRVIDEO_ENABLE_SSL" = 'true' ]; then
+  	dse_enable_ssl='true'
+
+	# The reference to this file is provided via a volume enabled
+	# on the dse-config container within docker-compose.yaml
+	# in the killrvideo-docker-common repo 
+  	dse_ssl_certfile='/opt/killrvideo-data/cassandra.cert'
```

Comment:
why killrvideo-data instead of killrvideo-dse-config? Not that it matters, just wondering about consistent naming.

Reply:
/opt/killrvideo-data is the path that dse-config uses locally within the Docker container for all of its other files so I am using the same path for consistency.  I did the same thing with all of the other affected services like /opt/killrvideo-generator for the generator.  These paths were already in place before I made any changes.  I could make the change easily enough if you would like.  Will need to retest.